### ISSUE TITLE
Add offscreen heading on a resource page

### ISSUE THREAD

**Issue by sinabahram** [benetech/ImageShare#231]
Where we have the metadata e.g. the subject and keywords, that dl does not have a heading before it. Can we put an sr-only heading of Metadata please, and then use aria-labelledby on the dl to point to said sr-only heading. Metadata heading should be an h2.

**Comment by clapierre** [comment]
Thanks @sinabahram for being so thorough.

**Comment by jkva** [comment]
Implemented via 667c235b4d9ab5064ab099461a66dd2a7b24bece

**Comment by jscholes** [comment]
@jkva The metadata list isn't `aria-labelledby` the new heading.  Also, @sinabahram not sure if we made a determination either way re: heading and/or list labels for individual file metadata?  E.g. on http://imgsdev.wpengine.com/btis_resource/biocomm-biology-in-a-community-context-dcmp/, the individual file has a type of "Video", format of "Streaming", etc.

**Comment by jkva** [comment]
Ah, missed that. Thanks 👍

**Comment by jkva** [comment]
Associated via 5d11e4a2b0d30a58376f04c8babb54c340fcc597

**Comment by sinabahram** [comment]
Awesome!
